packages/frontend/src/contexts/game/index.js

```diff
@@ -1,7 +1,7 @@
 import { createContext } from 'react';
 
 export const initialGameState = {
-  stage: 'playing', // setup, playing, complete
+  stage: 'PLAYING', // setup, playing, complete
```

Comment:
The `stage` value could come from the `GAME_STAGES` object everywhere it's referenced.